site/ja/install/gpu.md

```diff
@@ -0,0 +1,196 @@
+# GPUサポート
+
+TensorFlowのGPUサポートには、さまざまなドライバーとライブラリが必要です。
+インストールを簡単にし、ライブラリの競合を避けるために、
+[GPUサポート付きのTensorFlowのDockerイメージ](./docker.md)を使用することをおすすめします（Linuxのみ）。
+このセットアップには[NVIDIA® GPU drivers](https://www.nvidia.com/drivers){:.external}のみが必要です。
+
+これらのインストール手順はTensorFlowの最新リリース用です。
+古いTensorFlowリリースで使用する場合、CUDAおよびcuDNNバージョン用の
+[テスト済みビルド設定](./source.md#linux)を参照してください。
+
+## pipパッケージ
+
+利用可能なパッケージ、システム要件、および手順については、
+[pipインストールガイド](./pip)を参照してください。
+GPUサポート付きTensorFlowパッケージを`pip`インストールするには、安定版または開発版パッケージを選択してください:
+
+<pre class="prettyprint lang-bsh">
+<code class="devsite-terminal">pip install tensorflow-gpu  # stable</code>
+
+<code class="devsite-terminal">pip install tf-nightly-gpu  # preview</code>
+</pre>
+
+### TensorFlow 2.0 Beta
+
+GPUサポートのテスト用に[TensorFlow 2.0 Beta](https://www.tensorflow.org/beta)が利用可能です。
+インストールするには:
+
+<pre class="prettyprint lang-bsh">
+<code class="devsite-terminal">pip install tensorflow-gpu==2.0.0-beta0</code>
```

Comment:
Original source changed:

beta0 -> beta1

Reply:
I didn't notice it. Thanks!